CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Plugin support authenticating with access token provided by authn-k8s
```

Comment:
```suggestion

### Added
- Plugin supports authenticating with access token provided by authn-k8s.
```

Reply:
i think we should omit the `provided by authn-k8s` as it will work with any Conjur access token. I would use `Plugin supports authenticating with a Conjur access token`